server/app/services/rpc_server.rb

```diff
@@ -25,8 +27,27 @@ def initialize(code, message, backtrace = nil)
 
   attr_reader :handlers
 
-  def initialize
+  # @param [SizedQueue] queue
+  def initialize(queue)
+    @queue = queue
     @handlers = {}
+    @counter = 0
+    @processing = false
+  end
+
+  def process!
+    sleep 0.1 # avoid busy loop block in actor
```

Comment:
I still don't get it... comments should make the code less confusing, not more confusing :D 

Is that sleep supposed to be inside the `while` loop?